src/MongoDB.Driver/Encryption/EncryptOptions.cs

```diff
@@ -0,0 +1,98 @@
+﻿/* Copyright 2019-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using MongoDB.Driver.Core.Misc;
+
+namespace MongoDB.Driver.Encryption
+{
+    /// <summary>
+    /// Encryption options for explicit encryption.
+    /// </summary>
+    public class EncryptOptions
+    {
+        // private fields
+        private readonly string _algorithm;
+        private readonly string _alternateKeyName;
+        private readonly byte[] _keyId;
+
+        // constructors
+        /// <summary>
+        /// Initializes a new instance of the <see cref="EncryptOptions"/> class.
+        /// </summary>
+        /// <param name="algorithm">The encryption algorithm.</param>
+        /// <param name="alternateKeyName">The alternate key name.</param>
+        /// <param name="keyId">The keyId.</param>
```

Comment:
Perhaps a small description of what the keyId should be?
https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/client-side-encryption.rst#keyid